bindings/python/src/error_code.cpp

```diff
@@ -64,7 +64,7 @@ namespace {
 	struct ec_pickle_suite : boost::python::pickle_suite
 	{
 		static boost::python::tuple
-		getinitargs(error_code const& ec)
+		getinitargs(error_code const& /*ec*/)
```

Comment:
the typical way of dealing with these warnings is to just delete the argument name

Reply:
Pardon, but this is precisely what is done here, no?

Reply:
oh, I meant deleted instead of just commented out. not a big deal. I have a rule-of-thumb, if a parameter is difficult to understand what it does when its name is deleted, it should probably have a more specific type